docs/tutorial2023_unfolding/unfolding_exercise.md

```diff
@@ -0,0 +1,251 @@
+
+```
+
+where you can see the contributions from various background processes, namely Z+jets, $t\bar{t}$ and the single top, as well as signal processes corresponding to the STXS scheme discussed above. 
+
+One of the most important stages in the analysis design, is to make sure that the reco-level categories are pure with the corresponding gen-level processes.
+
+To explicitly check it, one can plot the contributions of gen-level bins in all of the reco-level bins. We propose to use the script provided in the tutorial git-lab page. 
+
+```shell
+python scripts/get_migration_matrix.py counting/combined_ratesOnly.txt
+
+```
+![](figures/migration_matrix_zh.png) 
+
+Now that we checked the response matrix we can attempt the maximum likelihood unfolding. We can use the `multiSignalModel` physics model available in `Combine`. To prepare the workspace we can run the following command: 
+
+```shell
+text2workspace.py -m 125  counting/* -P HiggsAnalysis.CombinedLimit.PhysicsModel:multiSignalModel  --PO verbose --PO 'map=.*/.*ZH_lep_PTV_75_150_hbb:r_zh_75_150[1,-5,5]' --PO 'map=.*/.*ZH_lep_PTV_150_250_0J_hbb:r_zh_150_250noj[1,-5,5]'  --PO 'map=.*/.*ZH_lep_PTV_150_250_GE1J_hbb:r_zh_150_250wj[1,-5,5]' --PO 'map=.*/.*ZH_lep_PTV_250_400_hbb:r_zh_250_400[1,-5,5]' --PO 'map=.*/.*ZH_lep_PTV_GT400_hbb:r_zh_gt400[1,-5,5]' -o ws_counting.root
+```
+where we use `--PO 'map=bin/process:poi[init, min, max]'`. So in the example above a signal POI is assigned to each gen-level bin independent on reco-level bin. This allows to take into account the non-trivial acceptance effects. One can also perform bin-by-bin unfolding using the mapping to the bin names rather that processes, e.g. `'map= vhbb_Zmm_gt400_13TeV/.*:r_reco_zh_gt400[1,-5,5]'`, but this method is not recommended and can be used for tests.
+
+To extract the measurement let's run the initial fit first: 
+
+```shell
+combineTool.py -M MultiDimFit --datacard ws_counting.root --setParameters r_zh_250_400=1,r_zh_150_250noj=1,r_zh_75_150=1,r_zh_150_250wj=1,r_zh_gt400=1 --redefineSignalPOIs r_zh_75_150,r_zh_150_250noj,r_zh_150_250wj,r_zh_250_400,r_zh_gt400 --saveWorkspace -t -1 --X-rtd FAST_VERTICAL_MORPH  
+```
+
+While the uncertainties on the POI can be extracted in multiple ways, the most robust way is to run the likelihood scans for POI corresponding to each gen-level bin, it allows to spot discontinuities in the likelihood shape in case of the problems with the model. 
+
+```shell
+combineTool.py -M MultiDimFit --datacard ws_counting.root -t -1 --setParameters r_zh_250_400=1,r_zh_150_250noj=1,r_zh_75_150=1,r_zh_150_250wj=1,r_zh_gt400=1 --redefineSignalPOIs r_zh_75_150,r_zh_150_250noj,r_zh_150_250wj,r_zh_250_400,r_zh_gt400 --saveWorkspace --algo=grid --points=100 -P r_zh_75_150 --floatOtherPOIs=1 -n scan_r_zh_75_150
+
+```
+Now we can plot the likelihood scan and extract the expected intervals.
+
+```shell
+python scripts/plot1DScan.py higgsCombinescan_r_zh_75_150.MultiDimFit.mH120.root -o r_zh_75_150 
```

Comment:
Is this different than the `plot1DScan.py` in the official repo? Maybe we should use that one directly, so they know its there and not just part of the tutorial repo?

I think it is also missing the `--POI r_zh_75_150` argument.